python/jsbeautifier/javascript/beautifier.py

```diff
@@ -561,6 +574,9 @@ def handle_start_expr(self, current_token):
                 current_token, current_token.newlines)
 
         self.print_token(current_token)
+        if current_token.text == '[' and \
+                self._options.brace_style == "expand-all":
+            self.print_newline()
```

Comment:
New feature to put starting square bracket on its own separate line (Newline afterwards)